packages/svelte/src/performance.ts

```diff
@@ -39,23 +34,24 @@ export function trackComponent(options?: TrackComponentOptions): void {
 
   let initSpan: Span | undefined = undefined;
   if (mergedOptions.trackInit) {
-    initSpan = recordInitSpan(transaction, componentName);
+    initSpan = recordInitSpan(componentName);
   }
 
   if (mergedOptions.trackUpdates) {
     recordUpdateSpans(componentName, initSpan);
   }
 }
 
-function recordInitSpan(transaction: Transaction, componentName: string): Span {
-  // eslint-disable-next-line deprecation/deprecation
-  const initSpan = transaction.startChild({
+function recordInitSpan(componentName: string): Span | undefined {
+  const initSpan = startInactiveSpan({
+    onlyIfParent: true,
     op: UI_SVELTE_INIT,
     name: componentName,
-    origin: 'auto.ui.svelte',
+    attributes: { [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: 'auto.ui.svelte' },
   });
 
   onMount(() => {
+    if (!initSpan) return;
```

Comment:
we can remove this again, as a recent PR changed the signature of `startInactiveSpan` to always return a span :)